community.md

```diff
@@ -0,0 +1,7 @@
+# Community Web Integrations
+
+## [Shadertoy on Kraken](https://github.com/robbieplata/shadertoy-on-kraken)
```

Comment:
We should give folks a way to market their actual custom web integration urls - especially since many can be serverless.

How about something like this for readability:

## Shadertoy on Kraken

> Overlays processor temperatures as a gauge over an iframe of community-made shaders from shadertoy.com. With this project, you can monitor your CPU performance while enjoying the art of shader programming.

| Author | Documentation | Web Integration Url |
| :------- | :---------------- | :---------------------- |
 [@robbieplata](https://github.com/robbieplata) | [robbieplata/shadertoy-on-kraken](https://github.com/robbieplata/shadertoy-on-kraken) | N/A, Run locally or self host |

---

## My web integration

> This is the description of my web integration. It has features such as light speed, red magic, and ultrahand movement/physics/attaching.

| Author | Documentation | Web Integration Url |
| :------- | :---------------- | :---------------------- |
 [@foo](https://github.com/foo) | [foo/bar](https://github.com/foo/bar) | https://foo.github.io/bar/ |

---

```markdown
// code

## Shadertoy on Kraken

> Overlays processor temperatures as a gauge over an iframe of community-made shaders from shadertoy.com. With this project, you can monitor your CPU performance while enjoying the art of shader programming.

| Author | Documentation | Web Integration Url |
| :------- | :---------------- | :---------------------- |
 [@robbieplata](https://github.com/robbieplata) | [robbieplata/shadertoy-on-kraken](https://github.com/robbieplata/shadertoy-on-kraken) | N/A, Run locally or self host |

---

## My web integration

> This is the description of my web integration. It has features such as light speed, red magic, and ultrahand movement/physics/attaching.

| Author | Documentation | Web Integration Url |
| :------- | :---------------- | :---------------------- |
 [@foo](https://github.com/foo) | [foo/bar](https://github.com/foo/bar) | https://foo.github.io/bar/ |

---
```